src/main/java/stream/flarebot/flarebot/commands/moderation/mod/BanCommand.java

```diff
@@ -30,22 +30,22 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     return;
                 }
                 Member target = guild.getGuild().getMember(user);
-                if(target == null) {
-                    MessageUtils.sendErrorMessage("That user is not on this server, you could try to forceban and try with the ID", 
-                        channel, sender);
+                if (target == null) {
+                    MessageUtils.sendErrorMessage("That user is not on this server, you could try to forceban and try with the ID",
+                            channel, sender);
                     return;
                 }
                 String reason = null;
                 if (args.length >= 2)
                     reason = MessageUtils.getMessage(args, 1);
                 try {
                     channel.getGuild().getController().ban(target, 7, reason).queue();
-                    guild.getAutoModConfig().postToModLog(user, sender, new Punishment(Punishment.EPunishment.BAN), reason);
+                    guild.getAutoModConfig().postToModLog(user, sender, ModlogAction.BAN.toPunishment(), reason);
                     channel.sendMessage(new EmbedBuilder()
                             .setColor(Color.GREEN)
                             .setDescription("The ban hammer has been struck on " + user.getName() + " \uD83D\uDD28")
                             .setImage(channel.getGuild().getId().equals(FlareBot.OFFICIAL_GUILD) ?
-                                    "https://cdn.discordapp.com/attachments/229704246004547585/361184232128905216/Webp.net-resizeimage.png" : null)
+                                    "https://flarebot.stream/img/banhammer.png" : null)
```

Comment:
won't we run into cloudflare problems with this?

Reply:
Nope because it's not us who requests it, it's the client